client/client.go

```diff
@@ -3039,10 +3039,15 @@ func (c *Client) setGaugeForCPUStats(nodeID string, hStats *stats.HostStats, bas
 			Value: cpu.CPU,
 		})
 
-		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "total"}, float32(cpu.Total), labels)
+		// Keep "total" around to remain compatible with older consumers of the metrics
+		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "total"}, float32(cpu.TotalPercent), labels)
+		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "total_percent"}, float32(cpu.TotalPercent), labels)
+		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "total_ticks"}, float32(cpu.TotalTicks), labels)
 		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "user"}, float32(cpu.User), labels)
 		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "idle"}, float32(cpu.Idle), labels)
 		metrics.SetGaugeWithLabels([]string{"client", "host", "cpu", "system"}, float32(cpu.System), labels)
+
+		metrics.IncrCounterWithLabels([]string{"client", "host", "cpu", "counter_ticks"}, float32(cpu.TotalTicks), labels)
```

Comment:
The name "total" for the `total_ticks` and `total_percent` is unfortunate because it's the "total" as in "across all types of CPU resources" whereas this is also a counter where "total" means "across all time". To make the relationship between the gauge and the counter clear, what do you think about naming this `total_ticks_count` like so:

```suggestion
		metrics.IncrCounterWithLabels([]string{"client", "host", "cpu", "total_ticks_count"}, float32(cpu.TotalTicks), labels)
```

That'll make `client.host.cpu.total_ticks` and `client.host.cpu.total_ticks_count` fall together in all the documentation.

(Same suggestion applies everywhere)

Reply:
Excellent idea :)